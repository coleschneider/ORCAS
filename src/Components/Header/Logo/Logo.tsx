import React from 'react';
import ImageLogo from 'Images/logo.svg';

const Logo = () => (
  <div className="header-logo">
    <div className="site-logo">
      <img src={ImageLogo} alt="Homepage" />
    </div>
  </div>
);

export default Logo;
