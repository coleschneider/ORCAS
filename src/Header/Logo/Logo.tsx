import React from 'react';
import ImageLogo from '../../images/logo.svg';

const Logo = () => (
  <div className="header-logo">
    <div className="site-logo">
      <img src={ImageLogo} alt="Homepage" />
    </div>
  </div>
);

export default Logo;
