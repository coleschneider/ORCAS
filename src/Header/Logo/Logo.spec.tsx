import * as React from 'react';
import Logo from './Logo';

describe('Logo', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });
});
