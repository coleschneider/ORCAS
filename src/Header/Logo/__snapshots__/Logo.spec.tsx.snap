// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should match the snapshot 1`] = `
<div
  className="header-logo"
>
  <div
    className="site-logo"
  >
    <img
      alt="Homepage"
      src="logo.svg"
    />
  </div>
</div>
`;
